bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html

```diff
@@ -0,0 +1,347 @@
+          dtSuffix
+          ktbDatetimePicker
+          [timeEnabled]="true"
+          [secondsEnabled]="true"
+          (selectedDateTime)="evaluationFormData.timeframeStart = $event"
+        >
+          <dt-icon name="calendar"></dt-icon>
+        </button>
+      </dt-form-field>
+    </div>
+
+    <div
+      fxLayout="row"
+      class="mb-3"
+      fxLayoutGap="15px"
+      *ngIf="evaluationFormData.evaluationType === TRIGGER_EVALUATION_TIME.START_END"
+    >
+      <dt-form-field class="overlay-origin">
+        <dt-label class="required">Start at</dt-label>
+        <div class="input-overlay" (click)="startDateButton?._elementRef?.nativeElement.click()"></div>
+        <input dtInput [(ngModel)]="evaluationFormData.startDatetime" [errorStateMatcher]="showErrorStateMatcher" />
+        <button
+          dt-icon-button
+          dtSuffix
+          variant="nested"
+          [disabled]="!evaluationFormData.startDatetime"
+          (click)="evaluationFormData.startDatetime = undefined"
+        >
+          <dt-icon name="abort"></dt-icon>
+        </button>
+        <button
+          #startDateButton
+          dt-icon-button
+          dtSuffix
+          ktbDatetimePicker
+          uitestid="keptn-trigger-button-starttime"
+          [timeEnabled]="true"
+          [secondsEnabled]="true"
+          (selectedDateTime)="evaluationFormData.startDatetime = $event"
+        >
+          <dt-icon name="calendar"></dt-icon>
+        </button>
+        <dt-hint>Has to be before End at</dt-hint>
+      </dt-form-field>
+      <dt-form-field class="overlay-origin">
+        <dt-label class="required">End at</dt-label>
+        <div class="input-overlay" (click)="endDateButton?._elementRef?.nativeElement.click()"></div>
+        <input dtInput [(ngModel)]="evaluationFormData.endDatetime" [errorStateMatcher]="showErrorStateMatcher" />
+        <button
+          dt-icon-button
+          dtSuffix
+          variant="nested"
+          [disabled]="!evaluationFormData.endDatetime"
+          (click)="evaluationFormData.endDatetime = undefined"
+        >
+          <dt-icon name="abort"></dt-icon>
+        </button>
+        <button
+          #endDateButton
+          dt-icon-button
+          dtSuffix
+          ktbDatetimePicker
+          uitestid="keptn-trigger-button-endtime"
+          [timeEnabled]="true"
+          [secondsEnabled]="true"
+          (selectedDateTime)="evaluationFormData.endDatetime = $event"
+        >
+          <dt-icon name="calendar"></dt-icon>
+        </button>
+        <dt-hint>Has to be after Start at</dt-hint>
+        <dt-error
+          *ngIf="
+            evaluationFormData.startDatetime &&
+            evaluationFormData.endDatetime &&
+            !checkStartEndValidity(evaluationFormData.startDatetime, evaluationFormData.endDatetime)
+          "
+          uitestid="keptn-trigger-evaluation-date-error"
+          >Start date must be before end date and vice versa.</dt-error
```

Comment:
"vice versa" is not logical here. I think we can drop this.
```suggestion
          >Start date must be before end date.</dt-error
```